client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nimiq/network-client",
-  "version": "0.1.7",
+  "version": "0.1.8-beta.1",
```

Comment:
Does it need to be beta?

Reply:
No, of course not, just waited for approval and now I'm gonna make it final.